### PR TITLE
feature/552_docker_readme

### PR DESCRIPTION
**Pull request by hermanjunge**

### CODE REVIEW

**Comment by hermanjunge** [comment]
@fgalan

File: docker/README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # Cygnus Containerization
 
-The process of building this application and packing it into a container could be a little long in time terms. Addressing this issue, the process is separated in two steps: Building the jar and packing the cygnus container.
+The process of building this application and packing it into a container could be a little long in time terms. Addressing this issue the process is separated in two steps: Building the _jar_ and packing the cygnus container.
```
Comment (fgalan) [inline docker/README.md]: packing -> packaging ?

Comment (hermanjunge) [inline docker/README.md]: No. `packing it`

Comment (fgalan) [inline docker/README.md]: Ok. NTC

**Comment by fgalan** [comment]
LGTM (passing the ball to @frbattid for final LGTM + merge + delete branch)

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
 - [HARDENING] Cygnus user creation added to the from the sources installation guide (#558)
 - [FEATURE] Add a binary implementation of the HDFS backend (#537)
+- [FEATURE] Add docker containerization (#552)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: I would prefer:

    - [HARDENING] Add docker as an option in the README for installing Cygnus (#552)

**Comment by frbattid** [comment]
There is a documentation folder at `/doc/installation` that is intended to hold all the installation options regarding Cygnus. I will simply add a piece of documentation there with a link to the `/docker/README.md`.

By having that and fixing the line within CHANGES_NEXT_RELEASE, LGTM for me. Thanks @hermanjunge !

**Comment by hermanjunge** [comment]
Done with the changes.

**Comment by frbattid** [comment]
LGTM!

**Pull request merged**